app/clients/freshdesk.py

```diff
@@ -80,7 +80,7 @@ def _generate_ticket(self) -> Dict[str, Union[str, int, List[str]]]:
     def send_ticket(self) -> int:
         try:
             api_url = current_app.config["FRESH_DESK_API_URL"]
-            if not api_url:
+            if not api_url or current_app.config["FRESH_DESK_ENABLED"] is False:
```

Comment:
what happens in this case? Does this throw a warning in #notification-ops?
I'm not sure if we want to raise an error for the second half - but maybe just a continue?

Reply:
We can definitely add tests for this! 
I wasn't sure about the placement of the check here. Originally I had it wrapping the post and just returning a 201 if it was disabled, since the implementation is disabled, it also seemed appropriate to throw a `NotImplementedError`. I didn't see any messages in #notification-ops but if that's a possibility then we should avoid it. This type of flag is only something we'd use in development correct, we wouldn't see use in prod?

Reply:
ya if you can add a test that would be great, if its cumbersome dw about it